docs/book/src/topics/reference/reference.md

```diff
@@ -15,4 +15,6 @@
 | ExternalResourceGC            | EXP_EXTERNAL_RESOURCE_GC          | false   |
 | AlternativeGCStrategy         | EXP_ALTERNATIVE_GC_STRATEGY       | false   |
 | TagUnmanagedNetworkResources  | TAG_UNMANAGED_NETWORK_RESOURCES   | true    |
-| ROSA                          | EXP_ROSA                          | false   |
\ No newline at end of file
+| ROSA                          | EXP_ROSA                          | false   |
```

Comment:
Probably a naive question from my side. If you use ROSA do you disable AWSMachine and AWSCluster always?

Reply:
It doesn't make conflict with ROSA, but in production env you don't run controllers that you don't use.

Reply:
So really it's either the Rosa controllers or the original capa controllers then.

Reply:
Got it, thanks @serngawy .